modules/database/src/ioc/db/dbChannel.h

```diff
@@ -60,6 +62,9 @@ typedef struct dbChannel {
     ELLLIST filters;          /* list of filters as created from JSON */
     ELLLIST pre_chain;        /* list of filters to be called pre-event-queue */
     ELLLIST post_chain;       /* list of filters to be called post-event-queue */
+    /* Support for optimizing scalar requests */
+    fastConvert *getCvt;      /* fast get conversion function */
+    short lastGetdbrType;     /* last dbChannelGet dbrType */
```

Comment:
I find myself wondering if the speedup from caching `dbFastGetConvertRoutine[dbfType][dbrType]` is worth the added code complexity.

Reply:
I wondered, too. Dirk told us that he measured it and found it makes a great difference. I am too lazy right now to look up the message in the launchpad MR.